sky/shell/manifest.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_SHELL_MANIFEST_H_
+#define SKY_SHELL_MANIFEST_H_
+
+#include <string>
+
+#include "base/memory/scoped_ptr.h"
+#include "base/version.h"
+#include "url/gurl.h"
+
+namespace sky {
+namespace shell {
+
+struct Manifest {
```

Comment:
OK. Do you recommend moving the platform dependent code there, too, or keep all that in the toplevel android and ios dirs?